test/br_to_try.wasm.fromBinary

```diff
@@ -1,6 +1,6 @@
 (module
- (type $0 (func (param i32)))
- (type $1 (func))
+ (type $0 (func))
+ (type $1 (func (param i32)))
```

Comment:
Why did this change? I can't find a new `--preserve-type-order` that applies to it, and I assume without that nothing should change?

Reply:
All of the tools except for wasm-opt now preserve the type order unconditionally, so changes are possible in any test that uses a tool other than wasm-opt.